docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc

```diff
@@ -96,7 +95,7 @@ spec:
     name: postgres-creds
 ----
 
-IMPORTANT: Changing disk size after initial provisioning is currently not supported.
+NOTE: When changing the disk size from a plan, it will restart the redis pods. Any subsequent disk resizes should not trigger restarts.
```

Comment:
```suggestion
NOTE: Changing the disk size from a plan will restart the Redis pods. Any further disk size changes should not trigger a restart.
```

Reply:
I've modified your suggestion slightly, what do you think?

Reply:
approved